cornflow/schemas/case.py

```diff
@@ -51,3 +53,17 @@ class CaseFromInstanceExecution(Schema):
     name = fields.Str(required=True)
     description = fields.Str()
     path = fields.Str(required=True)
+
+
+class CaseToLive(Schema):
+    """ """
+
+    id = fields.Str(required=True)
+    schema = fields.Str(required=True)
+
+
+class CaseEditRequest(Schema):
+    name = fields.String()
+    description = fields.String()
+    schema = fields.String()
+    path = fields.Str()
```

Comment:
does it make sense to edit the `path` property here? in theory that should be done through the `move` endpoint, isn't it?